src/main/java/ploopy/Task.java

```diff
@@ -0,0 +1,135 @@
+package ploopy;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Stores name and date of task. Marks as complete or incomplete
+ *
+ */
+
+public abstract class Task {
```

Comment:
I like your use of inheritance by abstracting common methods of Events, Deadlines and ToDo into a class Task!